app/config.py

```diff
@@ -80,7 +80,23 @@ class Config(object):
     CBC_PROXY_ENABLED = True
     ENABLED_CBCS = {BroadcastProvider.EE, BroadcastProvider.THREE, BroadcastProvider.O2, BroadcastProvider.VODAFONE}
 
-    SQLALCHEMY_DATABASE_URI = "postgresql://postgres:root@localhost/emergency_alerts"
+    if os.environ.get("MASTER_USERNAME"):
+        print("Using master credentials for db connection")
+        SQLALCHEMY_DATABASE_URI = "postgresql://{user}:{password}@{host}:{port}/{database}".format(
+            user=os.environ.get("MASTER_USERNAME", "root"),
+            password=os.environ.get("MASTER_PASSWORD"),
+            host=os.environ.get("RDS_HOST", "localhost"),
+            port=os.environ.get("RDS_PORT", 5432),
+            database=os.environ.get("DATABASE", "emergency_alerts"),
+        )
+    else:
+        print("Using no credentials for db connection")
```

Comment:
It defaults to that, however it can now be overridden if required.  The new local dev environment uses docker containers extensively and in order to connect to the other containers it needs to use host.docker.internal as a domain name (both for the tests and for the connections).